.github/workflows/publish_connectors.yml

```diff
@@ -106,26 +138,21 @@ jobs:
           spec_cache_gcs_credentials: ${{ secrets.SPEC_CACHE_SERVICE_ACCOUNT_KEY_PUBLISH }}
           s3_build_cache_access_key_id: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
           s3_build_cache_secret_key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
-          subcommand: "connectors --concurrency=1 --execute-timeout=3600 --metadata-changes-only publish --main-release"
+          subcommand: "connectors --name ${{ matrix.connector }} --metadata-changes-only publish --main-release"
```

Comment:
if you know what this `--metadata-changes-only` flag does, lmk - I think we can remove it, but `Only test connectors with modified metadata files in the current branch.` isn't super clear